swift/apple/Makefile

```diff
@@ -137,3 +193,37 @@ tail-tunnel-log:
 	else \
 		echo "No tunnel log files found"; \
 	fi
+
+.PHONY: help
+help:
+	@echo "Available targets:"
+	@echo "  all                  - Build and install iOS/macOS app (default)"
+	@echo "  build                - Build iOS/macOS app with Rust backend (default: Debug)"
+	@echo "  install              - Install built app to /Applications and launch"
+	@echo "  clean                - Clean all build artifacts"
+	@echo "  format               - Format Swift code"
+	@echo "  setup                - Install required Rust targets"
+	@echo ""
+	@echo "Log viewing targets:"
+	@echo "  show-client-log      - View latest client log file (JSONL)"
+	@echo "  tail-client-log      - Follow latest client log file in real-time"
+	@echo "  show-client-log-pretty - View formatted client log (requires jq)"
+	@echo "  show-ext-log         - View network extension logs only (last 30min)"
+	@echo "  tail-ext-log         - Follow network extension logs only"
+	@echo "  show-all-logs        - View ALL Firezone logs from console (last 30min)"
+	@echo "  tail-all-logs        - Follow ALL Firezone logs in real-time"
+	@echo "  show-connlib-log     - View connlib log file (requires sudo)"
+	@echo "  tail-connlib-log     - Follow connlib log file (requires sudo)"
+	@echo "  show-tunnel-log      - View latest tunnel log file (requires sudo)"
+	@echo "  tail-tunnel-log      - Follow latest tunnel log file (requires sudo)"
+	@echo ""
+	@echo "Variables:"
+	@echo "  CONFIGURATION  - Build configuration (Debug/Release, default: Debug)"
+	@echo ""
+	@echo "Examples:"
+	@echo "  make                              # Build and install (Debug)"
+	@echo "  make CONFIGURATION=Release        # Build and install (Release)"
+	@echo "  make build-macos                  # Build only, don't install"
+	@echo "  make install                      # Install previously built app"
+	@echo "  make show-client-log              # View latest app log file"
+	@echo "  make show-ext-log                 # View network extension logs"
```

Comment:
This is prone to get outdated I reckon but happy to keep it in here if you want it.

Reply:
ideally, we'd move to `just` or `mise` at some point, so this wouldn't be needed at all, as it's generated on-the-fly. 

if you don't mind, I'll keep it for a bit as it can be helpful when we do the change.